### PR TITLE
SqlTest improvements

### PR DESCRIPTION
**Pull request by Kielek**
## Why

Related to #970

## Tests

Simplify assertions.
Change connection string: `localhost` --> `127.0.0.1`
Cleanup code.
Execute unit tests before integration tests.

## Checklist

- ~~[ ] `CHANGELOG.md` is updated.~~
- ~~[ ] Documentation is updated.~~
- [x] New features are covered by tests.

### CODE REVIEW

**Comment by Kielek** [comment]
On my fork, similar changes passed 10/10 (with changed localhost to 127.0.0.1).

**Review by rajkumar-rangaraj — APPROVED** [review]
LGTM

File: test/IntegrationTests/SqlClientTests.cs [hunk 1]
```diff
@@ -55,12 +55,7 @@ public void SubmitTraces()
 
                 foreach (var span in spans)
                 {
-                    span.Service.Should().Be(ServiceName);
-                    span.Name.Should().Be("master");
-                    span.Tags.Should().Contain(new KeyValuePair<string, string>("db.system", "mssql"));
-                    span.Tags.Should().Contain(new KeyValuePair<string, string>("db.name", "master"));
-                    span.Tags.Should().Contain(new KeyValuePair<string, string>("peer.service", $"localhost,{_sqlServerFixture.Port}"));
```
Comment (zacharycmontoya) [inline test/IntegrationTests/SqlClientTests.cs:62]: Was there a reason for removing these? These seem like good tests to have to ensure the spans have the right information

Comment (Kielek) [reply test/IntegrationTests/SqlClientTests.cs:62]: I do not think that it is AutoInstrumentation responsibility to test it.
It should be covered by library tests. Here we should only make a sanity check that instrumentation is _somehow working_.,

**Pull request merged**